chain-network/Cargo.toml

```diff
@@ -7,6 +7,7 @@ license = "MIT OR Apache-2.0"
 
 [dependencies]
 async-trait = "0.1"
+bincode = { version = "1.2", optional = true }
```

Comment:
oh no... can't we just add the missing encoding format by hand here?

Reply:
Why, I'd rather write in another dependency than figure it out :)
What's the framing, anyway? Big endian marshalling?

Reply:
It literally takes more time for me to dig through `bincode` sources, because the format is not documented.

Reply:
And then there's [this](https://github.com/servo/bincode/blob/439b12dbcba903b7681879f96f932dc20b0a5115/src/config/int.rs#L182). I'd rather delegate it to the dependency.